packages/twenty-front/src/modules/workspace/components/WorkspaceInviteTeam.tsx

```diff
@@ -69,7 +71,6 @@ type FormInput = {
 
 export const WorkspaceInviteTeam = () => {
   const { enqueueSnackBar } = useSnackBar();
-  const [sendInviteLink] = useSendInviteLinkMutation();
```

Comment:
As you've noticed already `useSendInviteLinkMutation` is also used in the onboarding flow.
If we end up keeping the current implementation with the workspace schema, then we should probably introduce a dedicated hook for this. The other path is to come back to an implementation within the core schema and the existing sendInviteLinkMutation.